### PR TITLE
Fix: Accordion content blinking

### PR DESCRIPTION
**Pull request by truph01**
- This PR will fix the issue https://github.com/oblador/react-native-collapsible/issues/473. We also encountered that bug in our project https://github.com/Expensify/App/issues/43397
- Here is the result after the fix:

https://github.com/oblador/react-native-collapsible/assets/170341821/dd1d192a-2a3e-4b48-856a-da1e0e10c48e

### CODE REVIEW

**Comment by truph01** [comment]
@oblador Can you help review this PR once you have a chance? Thanks

File: Collapsible.js [hunk 1]
```diff
@@ -186,9 +186,12 @@ export default class Collapsible extends Component {
       animating,
     } = this.state;
     const hasKnownHeight = !measuring && (measured || collapsed);
-    const style = hasKnownHeight && {
+    const style = hasKnownHeight ? {
```
Comment (Pujan92) [inline Collapsible.js:189]: @truph01 Simply setting the styles by omitting the `hasKnownHeight` won't work?

```js
const style = {
  overflow: 'hidden',
  height: height,
}
```

**Comment by oblador** [comment]
Hi, thanks for your PR! When I tested there's still junk on Android (iOS looks fine from what I can tell):

https://github.com/oblador/react-native-collapsible/assets/378279/2f349c63-3ea1-4985-aac6-74d2410e1320

**Comment by truph01** [comment]
@Pujan92 Can you reproduce the [issue](https://github.com/oblador/react-native-collapsible/pull/475#issuecomment-2201289049) on Android after applying the fix? It works well from my side:


https://github.com/oblador/react-native-collapsible/assets/170341821/5e7921b0-b83e-4e18-a7d9-d2963a5da813

**Comment by Pujan92** [comment]
It works well for me too. @oblador Could you plz check one more time on your side?

| Before  | After Fix |
| ------------- | ------------- |
| <video src="https://github.com/oblador/react-native-collapsible/assets/14358475/51e20bfb-fc41-4503-9f89-917c9360e599" />  | <video src="https://github.com/oblador/react-native-collapsible/assets/14358475/e907bda5-383d-432a-a5bf-19a5407af12c" />  |

**Comment by Pujan92** [comment]
Tried the example code with the functional component and was not able to reproduce the mentioned bug for android.

https://github.com/oblador/react-native-collapsible/assets/14358475/015d66eb-780c-484f-834b-c0898f6822be

**Comment by truph01** [comment]
@oblador Can you help review the PR? Thanks

**Comment by truph01** [comment]
@oblador Any update on the PR review?

Thx

File: Collapsible.js (hunk 1 above)
Comment (oblador) [reply Collapsible.js]: Fixed in 13af1dd

**Review by oblador — APPROVED** [review]

**Comment by Pujan92** [comment]
Thanks @oblador for the review and merge :)

**Comment by VNDRN** [comment]
This broke initially open collapsibles

**Pull request merged**